packages/pds-ember/app/styles/pds/components/tab-nav/_top.scss

```diff
@@ -0,0 +1,26 @@
+// TOP Variant: tabs above content
+$_module: "PDS.Components.TabNav.Top";
+/* --- [debug] CONFIG: #{$_module} --- */
+@use "../../theme";
+@use "../link/__config" as Link;
+
+$borderColor: theme.$borderColor;
+$borderWidth: 1px;
+
+@mixin layout {
+  display: flex;
+  border-bottom: $borderWidth solid $borderColor;
+
+  // FIXME: how should this [over]flow with constrained dimensions?
```

Comment:
Is there either a design spec for this edge case, or an existing JIRA ticket to reference?

My concern would be that this important question could get lost, if it isn't captured elsewhere.

Reply:
See https://www.figma.com/file/B7yd8loYS7nA2TiQcJ8UtV?node-id=3771:6348#48254074

Reply:
also captured in PDS-196